services/orbits/orbits/schemas.go

```diff
@@ -0,0 +1,170 @@
+package orbits
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+    "github.com/leanspace/terraform-provider-leanspace/helper"
+	"github.com/leanspace/terraform-provider-leanspace/helper/general_objects"
+)
+
+var validIdealOrbitTypes = []string{"SSO", "POLAR", "LEO", "GEO", "MEO", "OTHER"}
+
+var orbitSchema = map[string]*schema.Schema{
+	"id": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"satellite_id": {
+		Type:         schema.TypeString,
+		Required:     true,
+		ForceNew:     true,
+		ValidateFunc: validation.IsUUID,
+	},
+	"name": {
+		Type:     schema.TypeString,
+		Required: true,
+	},
+	"ideal_orbit": {
+		Type:     schema.TypeList,
+		Optional: true,
+		MinItems: 1,
+		MaxItems: 1,
+		Elem: &schema.Resource{
+			Schema: idealOrbitSchema,
+		},
+	},
+	"tags": general_objects.KeyValuesSchema,
+	"created_at": {
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "When it was created",
+	},
+	"created_by": {
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "Who created it",
+	},
+	"last_modified_at": {
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "When it was last modified",
+	},
+	"last_modified_by": {
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "Who modified it the last",
+	},
+}
+
+var idealOrbitSchema = map[string]*schema.Schema{
+	"type": {
+		Type:     schema.TypeString,
+		Required: true,
+	},
+	"inclination": {
+		Type:     schema.TypeFloat,
+		Required: true,
+	},
+	"right_ascension_of_ascending_node": {
+	    Type:     schema.TypeFloat,
+	    Required: true,
+    },
+    "argument_of_perigee": {
+        Type:     schema.TypeFloat,
+        Required: true,
+    },
+    "altitude_in_meters": {
+        Type:     schema.TypeFloat,
+        Required: true,
+    },
+    "eccentricity": {
+        Type:     schema.TypeFloat,
+        Required: true,
+    },
+    "perigee_altitude_in_meters": {
+        Type:     schema.TypeFloat,
+        Required: true,
```

Comment:
I'm referring to the request body in https://docs.leanspace.io/docs/api/branches/dev/orbits/operations/create-a-orbit

Reply:
ah you mean the create command request; these 3 fields are generated in the backend and sent back; I thought I should still declare them as required here. Was it a wrong assumption?

Reply:
ok right, I need to declare them as computed, got it

Reply:
If the user shall not provide this information but the backend computes them (which seem to be the case), it should be `Computed: true` instead of `Required: true`
If you need to provide it in some way (when updating/deleting/...) I think you need to can put it as `Optional: true`

Reply:
done :)